pkg/app/piped/planner/ecs/ecs.go

```diff
@@ -81,6 +81,14 @@ func (p *Planner) Plan(ctx context.Context, in planner.Input) (out planner.Outpu
 		return
 	}
 
+	// Force to use pipeline when the alwaysUsePipeline field was configured.
```

Comment:
nits, how about moving the check to perform QuickSync in case of no pipeline configured at line [L101](https://github.com/pipe-cd/pipe/pull/2472/files#diff-7c63cdce360ea81edee09d171ff98831074434fef3a1184f68b0000c6fabeba8R101) above this block and remove `cfg.Pipeline != nil` check?

Reply:
Sure. Nice catch! 👍

Reply:
@khanhtc1202 Applied your suggestion!

Reply:
Thanks 👍